hack/verify-api-reference-docs.sh

```diff
@@ -26,15 +26,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 kube::golang::setup_env
 
 API_REFERENCE_DOCS_ROOT="${KUBE_ROOT}/docs/api-reference"
-# Use REPO_DIR if provided so we can set it to the host-resolvable path
-# to the repo root if we are running this script from a container with
-# docker mounted in as a volume.
-# We pass the host output dir to update-api-reference-docs.sh, but use
-# the regular one to compute diff (they will be the same if running this
-# test on the host, potentially different if running in a container).
-REPO_DIR=${REPO_DIR:-"${KUBE_ROOT}"}
-HOST_OUTPUT_DIR="${REPO_DIR}/_tmp/api-reference"
-TMP_OUTPUT_DIR="${KUBE_ROOT}/_tmp/api-reference"
+HOST_OUTPUT_DIR="${KUBE_ROOT}/_tmp/api-reference"
+mkdir -p ${HOST_OUTPUT_DIR}
```

Comment:
I would recommend calling this something else, since it's not a host resolvable path. OUTPUT_DIR is fine.